Marlin/src/Marlin.cpp

```diff
@@ -944,7 +944,8 @@ void setup() {
   ui.init();
   ui.reset_status();
 
-  #if HAS_SPI_LCD && ENABLED(SHOW_BOOTSCREEN)
+  // (For MKS_MINI_12864 the bootscreen is shown in ui.init)
+  #if HAS_SPI_LCD && ENABLED(SHOW_BOOTSCREEN) && DISABLED(MKS_MINI_12864)
```

Comment:
It's better to have a conditional (defined in `Conditionals_LCD.h`) if this might be a common issue with quirky displays that run afoul of the the common standards. So, rather than having this LCD-specific condition…
```cpp
#if HAS_SPI_LCD && ENABLED(SHOW_BOOTSCREEN) && DISABLED(MKS_MINI_12864)
```
…we would have this instead…
```cpp
#if HAS_SPI_LCD && ENABLED(SHOW_BOOTSCREEN) && ENABLED(BOOTSCREEN_IN_SETUP)
```